tests/integration/test_connect.py

```diff
@@ -0,0 +1,36 @@
+import pytest
+from influxdb import InfluxDBClient
+from playhouse.postgres_ext import PostgresqlExtDatabase
+from pymongo import MongoClient
+
+from robotoff import settings
+
+
+@pytest.mark.integtest
+def test_connect_mongodb():
+    client = MongoClient(settings.MONGO_URI)
+    client.server_info()
+
+
+@pytest.mark.integtest
+def test_connect_postgres():
+    client = PostgresqlExtDatabase(
+        settings.POSTGRES_DB,
+        user=settings.POSTGRES_USER,
+        password=settings.POSTGRES_PASSWORD,
+        host=settings.POSTGRES_HOST,
+        port=5432,
+    )
+    client.connect()
+
+
+@pytest.mark.integtest
+def test_connect_influxdb():
+    client = InfluxDBClient(
+        settings.INFLUXDB_HOST,
+        settings.INFLUXDB_PORT,
+        settings.INFLUXDB_USERNAME,
+        settings.INFLUXDB_PASSWORD,
+        settings.INFLUXDB_DB_NAME,
+    )
+    client.get_list_users()
```

Comment:
Do we really need to use pytest to run this kind of things.

As already said (sorry for repeating) I would better have them as sanity checks, without any need of pytest to run this.

Reply:
Python is ok, I just mean we do not need pytest.

Reply:
And also don't put them in tests/  which folder is not copied on prod image. Make a script directory or something like that.

Reply:
Not sure why pytest is not as appropriate as for unit tests for instance, it allows us to see individual results and know where it went wrong, compared to re-implementing those features in a non-pytest python-written test :/. 

For putting them in the `tests/` folder, that would allow to have `tests/unit`, `tests/integration`, `tests/health`, which seemed more consistent than having them in a separate folder

Reply:
hum, ok this can be, but you will need to add pytest and all needed dependencies on prod image.

I'm also very reluctant to run integrations test (even unit tests) on production, as they can be destructive. So I would prefer the corresponding folders not to be copied on production image.